Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	docker build --tag snapshot-testing . && docker run --rm snapshot-testing
```

Comment:
now we can just `make test` locally to run on linux. maybe this should be `make test-linux`?